src/country_index.c

```diff
@@ -158,8 +177,24 @@ country_cmpfunc(const void *a, const void *b)
 void
 country_indexer_init()
 {
-    geoip = GeoIP_new(GEOIP_STANDARD);
+    geoip = GeoIP_open_type(GEOIP_COUNTRY_EDITION, GEOIP_MEMORY_CACHE);
+    if (geoip == NULL) {
+        fprintf(stderr, "country_index: Error opening IPv4 Country DB. Make sure libgeoip's GeoIP.dat file is available.\n");
```

Comment:
That line is a clear error that should always be outputted if there is a problem and since this is a daemon then it should be `syslog()`.

Don't seem to be any consistency through out the code, might make some effort later on to clean up and sanitize all the output.

For now, change the lines I noted on and rebase then we are good to merge.